File: basic/python/object.py
class Car:
    # Properti
    type = "Fiat"
    model = "500"
    color = "white"
# Method
def start(self):
    print("ini method start")
def drive(self):
    print("ini method drive")
def brake(self):
    print("ini method brake")
def stop(self):
    print("ini method stop")

#akses object
# Membuat objek mobil
car = Car()
print(car.type)
print(car.color)
# Memanggil method
# car.start()
# car.drive()
# car.brake()
# car.stop()

File: javascript/part-2/object.js
var car = {
    // properti
    type: "Fiat",
    model: "500",
    color: "white",
    // method
    start: function(){
        console.log("ini method start");
        },
    drive: function(){
        console.log("ini method drive");
        },
    brake: function(){
        console.log("ini method brake");
        },
    stop: function(){
        console.log("ini method stop");
        }
        };

// akses Object
console.log(car.type);
//Result "Fiat"
console.log(car.color);
//Result "white"
car.start();
//Result "ini method start"
car.drive();
//Result "ini method drive"

// penggunaan keyword this
function Person(firstName, lastName){
    // properti
    this.firstName = firstName;
    this.lastName = lastName;
    // method
    this.fullName = function(){
        return `${this.firstName} ${this.lastName}`
}
this.showName = function(){
console.log(this.fullName());
}
}
var person1 = new Person("Muhar", "Dian");
var person2 = new Person("Petani", "Kode");
person1.showName();
person2.showName();